docs/source/resources/faq.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Frequently Asked Questions
+description: Common questions asked at each stage of GraphQL adoption
+---
+
+Everyone has questions about how to properly set up a GraphQL schema, but not all questions are alike. In different stages of development, different things matter. This guide will questions that people commonly have have at every step along the journey to GraphQL in production.
+
+## Learning GraphQL
+
+> You are just beginning to learn GraphQL. You're learning about syntax so you can run a query in a playground, you're learning what a schema is and how to connect it to your services with resolvers, etc.
```

Comment:
I didn't actually mean for you to put `etc`, I just wanted some similar examples. Can you please take that out and restructure these sentences?